.snyk

```diff
@@ -7,54 +7,4 @@ ignore:
         reason: We need to update to Vertx 4.x
         expires: 2023-07-01T05:00:40.733Z
         created: 2022-09-23T05:00:40.743Z
-   SNYK-JAVA-IONETTY-2812456:
-    - '*':
-        reason: We need to update to Vertx 4.x
-        expires: 2023-07-01T05:00:40.733Z
-        created: 2022-09-23T05:00:40.743Z
-   SNYK-JAVA-IONETTY-1042268:
-    - '*':
-        reason: We need to update to Vertx 4.x
-        expires: 2023-07-01T05:00:40.733Z
-        created: 2022-09-23T05:00:40.743Z
```

Comment:
Were these resolved with Vert.x 3.9.14?

Reply:
Yes. Even though they're on the 4.x line, they're still doing security fixes on the 3.x line (at least until the end of the year).